src/js/providers/html5.js

```diff
@@ -744,7 +744,11 @@ define([
 
         this.setContainer = function(containerElement) {
             _container = containerElement;
-            containerElement.appendChild(_videotag);
+            if (containerElement.firstChild) {
```

Comment:
@pajong there should be a commercial PR that does the same for HLS and DASH providers.

@johnBartos Please verify passing `null` or `undefined` as the first arg to `insertBefore` is OK in IE9.